ingestion/functions/parsing/brazil_rio_de_janeiro/rio_de_janeiro.py

```diff
@@ -0,0 +1,149 @@
+    '''
+    if entry['idade']:
+        return float(entry['idade'])
+
+    
+            
+def parse_cases(raw_data_file, source_id, source_url):
+    """
+    Parses G.h-format case data from raw API data.
+    
+    Only include cases which include a 'dt_evento' as confirmed date field. This loses ~400 cases out of 350k.
+    
+    Using residential municipality ('municipio_res') as case location, as its the only location field other than 
+    'bairro' (neighborhood), which is missing in ~2/5 cases and cannot be easily geocoded with mapbox.
+    
+    
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.DictReader(f,delimiter=';')
+        cases = []
+        for entry in reader:
+            if entry['classificacao'] and entry['dt_evento']:
+                notes = []
+                case = {
+                        "caseReference": {
+                            "sourceId": source_id,
+                            "sourceUrl": source_url
+                        },
+                        "events": [
+                        {
+                            "name": "confirmed",
+                            "dateRange":
+                            {
+                                "start": convert_date(entry["dt_evento"]),
+                                "end": convert_date(entry["dt_evento"])
+                            }
+                        }
+                    ],
+                        "location": convert_location(entry),
+                        "demographics": {
+                            "ageRange": {
+                                "start": convert_age(entry),
+                                "end": convert_age(entry)
+                            },
+                            "gender": convert_gender(entry["sexo"])
+                            }
+                    }
+                
+            if entry['dt_sintoma']:
+                case["symptoms"] = {
+                    "status": "Symptomatic",
+                }
+                case["events"].append({
+                    "name": "onsetSymptoms",
+                    "dateRange": {
+                        "start": convert_date(entry['dt_sintoma']),
+                        "end": convert_date(entry['dt_sintoma']),
+                    }
+                })
+                
+            if entry['evolucao'] == 'OBITO':
+                case["events"].append({
+                "name": "outcome",
+                "value": "Death",
+                "dateRange": {
+                    "start": convert_date(entry['dt_obito']),
+                    "end": convert_date(entry['dt_obito']),
+                    }})
+            else:
+                case["symptoms"] = {
+                    "status": "Asymptomatic",
+                }
+
+            if entry["sexo"]=='I':
+                notes.append(f'Gender not specified in case notes.')
+
+            if entry["bairro"]:
```

Comment:
The wording for the neighbourhood entry in notes is a bit different in other parsers (at least the ones I have written). We should probably agree on a way to standardize this?